server/routes/profile.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const router = express.Router();
+const auth = require('../middleware/verify-auth');
+
+const Profile = require('../models/Profile');
+const { update } = require('../models/user');
+const User = require('../models/user');
+
+// Returns current user profile based on ID
+router.get('/:user_id', auth, async (req, res) => {
+    try {
+    //find profile by user id and populate email
+    const profile = await Profile.findOne({ user: req.params.user_id }).populate('User', ['email']);
+
+    if(!profile) {
+        return res.status(400).json({ msg: "No profile found." });
+    }
+
+    res.json(profile);
+
+    } catch(err){
+    console.log(err.message);
+    if(err.kind == "ObjectId") {
+        return res.status(400).json({ msg: "Profile not found for this Id." });
+    }
+    res.status(500).send('Server Error');
+    }
```

Comment:
You want to take the id from req.body (which is added by the auth middleware). Otherwise, you can't guarantee that user_id belogs to the authed user